src/main/java/output/duke.txt

```diff
@@ -0,0 +1,2 @@
+[T][ ] 111
```

Comment:
perhaps you can include txt files in `.gitignore` files so that your storage file wouldn't be uploaded to public